packages/node/test/unit/scenario.spec.js

```diff
@@ -47,5 +47,5 @@ for await (const scenario of loadScenarios()) {
     continue
   }
 
-  test.serial(scenario.name ?? '(unknown scenario)', testScenario, scenario)
```

Comment:
Why do we want that to no longer be serial? Or why did we need it before?

Reply:
It may have been required at some point, but I was probably just mistaken. Anyway, it goes faster now.